arduino-builder/main.go

```diff
@@ -97,17 +97,8 @@ func (h *foldersFlag) String() string {
 }
 
 func (h *foldersFlag) Set(csv string) error {
-	var values []string
-	if strings.Contains(csv, string(os.PathListSeparator)) {
-		values = strings.Split(csv, string(os.PathListSeparator))
-	} else {
-		values = strings.Split(csv, ",")
-	}
-
-	for _, value := range values {
-		value = strings.TrimSpace(value)
-		*h = append(*h, value)
-	}
+	csv = strings.TrimSpace(csv)
+	*h = append(*h, csv)
```

Comment:
Should we also remove the TrimSpace here? With it, you cannot have use directory name ending or starting in a space (which is unlikely to be useful, but why not just allow *any* filename?). A shell will trim spaces anyway, unless someone takes the effort to escape them (in which case we shouldn't trim them again)?